features/shopcarts.feature

```diff
@@ -22,12 +22,19 @@ Scenario: List all shopcarts
 Scenario: Create a Shopcart
     When I visit the "Home Page"
     And I set the Shopcart "user_id" to "1"
-    And I click the "Add Products" button
-    And I add "1" "Apple" to the cart
+    And I click the "Add-Products" button
+    And I add "2" of Product "3" to the cart
     And I click the "Create" button
     Then I should see Shopcart "1" in the results
     And I should not see "Status Code: 409. Shopcart for user 1 already exits" in the form
 
+Scenario: Visit an existing Shopcart
```

Comment:
I also created this scenario to try to understand what was happening. Should we leave it here and consider this as the "retreive a shopcart" issue: 
https://github.com/nyu-devops-echo/shopcarts/issues/71
?
@radhikamattoo @jonjrodriguez

Reply:
Since we have a separate issue for retrieving a shopcart we should probs create a separate PR for it

Reply:
ok! I'll remove this.